src/ipaperftest/core/main.py

```diff
@@ -186,6 +197,7 @@ def main(
     provider="idmci",
     idmci_lifetime=8,
     auth_spread=0,
+    expected_result_type="no_errors",
```

Comment:
Is there a reason these two options aren't closer to each other? Is it sorted by type?

Reply:
Both are at the end of the arguments with/without default value lists. I can move `expected_result_type` to the top of its list to have them together if that's better.